ios_mcp/source/menu.c

```diff
@@ -89,15 +110,47 @@ static void waitButtonInput(void)
     }
 }
 
+static int isSystemUsingDebugKeyset(void)
```

Comment:
FYI there is a system call to check for debug mode.
I can just replace this myself at some point though. I wanted to clean up some things in the menu anyways.

Reply:
This is checking if the system is using debug keys in general, not prod vs. devel. It's used to determine if devmenu, SCT, and kiosk menu should be displayed in Set Coldboot Title.